playground/nextjs-pages/src/pages/_document.tsx

```diff
@@ -8,7 +8,7 @@ import {
 
 export default function Document(props: DocumentProps & DocumentHeadTagsProps) {
   return (
-    <Html lang="en">
+    <Html lang="en" data-color-scheme="light">
```

Comment:
What's the difference between `data-color-scheme` and `data-mui-color-scheme`?

Reply:
Not sure if they should be the same actually, in the App Router I had noticed that `data-mui-color-scheme` was the correct one but in the docs it for Pages Router it's just `data-color-scheme`, will look into it more deeply to make sure it's correct in both cases.